### DISCUSSION TITLE
HOw to handle event bubbling

### DISCUSSION THREAD

**Discussion by devthub** [chakra-ui/chakra-ui#7008] [Q&A]
How do you handle event bubbling when your IconButton is inside a parent component that has its own onClick?

**Comment by TinsFox** [comment]
This is my usage.
```tsx
onClick={(event) => {
	event.stopPropagation();
	event.preventDefault();
}}
```

**Reply by devthub** [reply]
I did try that on IconButton component, event is undefined.

**Reply by TinsFox** [reply]
> I did try that on IconButton component, event is undefined.

Can you share your full code?

**Reply by devthub** [reply]
Thanks, it's working now...
IconButton's onclick did returns an event. don't know why it's not happening last time.
